package.json

```diff
@@ -5,7 +5,7 @@
   "main": "server.js",
   "dependencies": {
     "blpapi": "^1.0.0",
-    "bluebird": "^2.8.2",
+    "bluebird": "^2.9.13",
```

Comment:
I would probably squash down to these 3 commits eventually but keep it in a single PR.